core/src/test/scala/org/broadinstitute/dsde/rawls/entities/local/CaseSensitivitySpec.scala

```diff
@@ -3,7 +3,7 @@ package org.broadinstitute.dsde.rawls.entities.local
 import akka.actor.ActorSystem
```

Comment:
Most of the change in this file is making it work with Quicksilver. I've added comments in places where some other change was needed.